### PR TITLE
Pagination css

### PR DESCRIPTION
**Pull request by matthewwrobel**
-refactored to client side pagination because of bogus server implementation
-refactored search
-added css

@jabrake @alizehrehman @tchitrakorn tagging everyone because Jorge might not have time for this today. No rush.

### CODE REVIEW

File: client/questions_answers/components/HelpfulReport.jsx [hunk 1]
```diff
@@ -6,17 +6,22 @@ const HelpfulReport = (props) => {
   let helpfulIds = JSON.parse(localStorage.getItem(props.content_type === 'question' ? 'helpfulQuestions' : 'helpfulAnswers'));
 
   if (props.reported) {
-    renderReportedLink = (<span> Reported </span>);
+    renderReportedLink = (<span> <i>Reported</i> | </span>);
   } else {
-    renderReportedLink = (<button onClick={props.handleReportButtonClick}> Report </button>);
+    renderReportedLink = ( <span>
```
Comment (jabrake) [inline client/questions_answers/components/HelpfulReport.jsx:11]: Out of curiosity why are you wrapping some elements in a <span> tag?

File: client/questions_answers/components/HelpfulReport.jsx [hunk 2]
```diff
@@ -6,17 +6,22 @@ const HelpfulReport = (props) => {
...
+      <button
+        className="qa-button-link" onClick={props.handleReportButtonClick}> Report </button>
+    </span>
+    );
   }
 
   if (helpfulIds.includes(props.content_id)) {
-    renderHelpfulLink = (<span> {`Helpful? Yes ${props.helpfulness}`}</span>);
+    renderHelpfulLink = (<span> {'Helpful?'} <i>Yes</i> {`(${props.helpfulness}) | `}</span>);
```
Comment (jabrake) [inline client/questions_answers/components/HelpfulReport.jsx:19]: Do you still need JSX if you're just rendering a string?

**Review by jabrake — APPROVED** [review]
Looks great! Just left a couple of questions but looks good to merge.

File: client/questions_answers/components/HelpfulReport.jsx (hunk 2 above)
Comment (matthewwrobel) [reply client/questions_answers/components/HelpfulReport.jsx:19]: Ha yeah. There were a couple cases where I needed a space before or after text and that's the way I got it to work. Probably not the best way to do that. I guess I just decided to keep going with it! I'll figure out a better solution and refactor.

File: client/questions_answers/components/HelpfulReport.jsx (hunk 1 above)
Comment (matthewwrobel) [reply client/questions_answers/components/HelpfulReport.jsx:11]: I can't remember. Maybe I was planning to add a class or something? I'll check that out. Thanks!

**Pull request merged**